bigchaindb/common/schema/transaction.yaml

```diff
@@ -205,19 +202,22 @@ definitions:
             type_id:
               type: integer
           description: |
-              Fulfillment of a condition_, or put a different way, this is a
-              payload that satisfies a condition in order to spend the associated
-              asset.
+              Fulfillment of an `Output.condition`_, or put a different way, this is a
+              payload that satisfies an output condition in order to spend
```

Comment:
Fixed in https://github.com/bigchaindb/bigchaindb/pull/925/commits/36d0ff519efce04b5703c1d9edd040345e25f021, related to https://github.com/bigchaindb/bigchaindb/pull/925#discussion_r92399996.